### PR TITLE
Jorja new lsv

### PR DESCRIPTION
**Pull request by JorjaF**
so here is Lowest Scoring Visitor method again with the look_up_Team_name too

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -94,5 +94,17 @@ def goals_per_team
     team_goals
   end
 
-  
+  def look_up_team_name(team_id)
```
Comment (andilovetto) [inline lib/stat_tracker.rb:97]: excited to implement the use of this helper method into the methods that i have written!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -94,5 +94,17 @@ def goals_per_team
...
+    team = @team_factory.teams.find do |team|
+      team_id == team[:team_id]
+    end
+    team[:team_name]
+  end
+
+  def lowest_scoring_visitor
+    lsv = @game_factory.games.min_by do |game|
```
Comment (andilovetto) [inline lib/stat_tracker.rb:105]: i love the min_by & max_by enumerators! huge helper in writing these methods!

**Review by andilovetto — APPROVED** [review]
nice work, jorja! this looks great

**Review by bserge13 — APPROVED** [review]
Clean and concise. Very nice work, and love the usage of the helper method.

**Pull request merged**